.github/actions/spelling/line_forbidden.patterns

```diff
@@ -19,6 +24,12 @@
 # s.b. greater than
 \bgreater then\b
 
+# s.b. into
+#\sin to\s
+
+# s.b. opt-in
+\sopt in\s
```

Comment:
This can have a fairly high false-positive rate. Disabling it is easy, just prefix w/ `#` (as above).

Reply:
I'll keep that in mind!